### DISCUSSION TITLE
Api to check if module is enabled

### DISCUSSION THREAD

**Discussion by vitalybrandes** [OrchardCMS/OrchardCore#9805] [Q&A] [answered]
Is there any existing API that check if some module is enabled?

**Comment by hishamco** [comment, marked as answer]
I think you can do that by extenion manager

**Reply by hishamco** [reply]
Also check `ShellFeaturesManagers.GetEnabledFeaturesAsync()`

**Comment by weirdyang** [comment]
var features = _shellDescriptor.Features.Select(feature => feature.Id).ToImmutableArray();

**Reply by jptissot** [reply]
Something like this: https://github.com/StatCan/StatCan.OrchardCore/blob/master/src/Themes/DigitalAdmin/Views/Layout.cshtml#L63
